packages/frontend/src/main/layouts/TheMain.vue

```diff
@@ -62,9 +61,9 @@
 </template>
 <script>
 import { gql } from '@apollo/client/core'
-import { mainUserDataQuery } from '@/graphql/user'
 import { useNavigationDrawerAutoResize } from '../lib/core/composables/dom'
 import { ref } from 'vue'
+import { useIsLoggedIn } from '../lib/core/composables/core'
```

Comment:
is this a typo? - `user` or `use`?

Reply:
"use", it's a common prefix for composables https://vuejs.org/guide/extras/composition-api-faq.html